setup.cfg

```diff
@@ -5,7 +5,3 @@ ignore = E133, E203, E226, E228, N802, N803, N806, N812, N813, N815, N816, W503
 exclude =
   */__init__.py,
   config/*.py
-
-[tool:pytest]
-addopts = --flake8
-flake8-ignore = E133 E203 E226 E228 N802 N803 N806 N812 N813 N815 N816 W503
```

Comment:
Why was this stuff removed?

Reply:
I don't think this is recommended or supported any more? There is a `flake8` config above this with the same rules anyway.